### DISCUSSION TITLE
How can i change size of model?

### DISCUSSION THREAD

**Discussion by salahhusa9** [wire-elements/modal#421] [General]
i want to change size of model

**Comment by v-stamenova** [comment]
Refer to this section of the documentation - [Changing modal properties](https://github.com/wire-elements/modal?tab=readme-ov-file#changing-modal-properties).

Don't forget to add the classes to your tailwind safelist - refer to [this section which is a little below the first one](https://github.com/wire-elements/modal?tab=readme-ov-file#building-tailwind-css-for-production)
